src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -234,14 +245,19 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
                         response.ontologies = response.ontologies.filter(onto => onto.attachedToProject !== Constants.SystemProjectIRI);
 
                         this.ontologiesMetadata = response;
+
+                        // notifies the user that the select project has got 0 data model yet defined.
+                        if (!this.selectOntologyComponent && response.ontologies.length === 0) {
+                            this.errorMessage = 'No data model defined for the select project.';
+                        }
                     },
                     (error: ApiResponseError) => {
                         console.error(error);
                     }
                 );
             }
         } else {
-            this.errorMessage = 'You are not part of any project.';
+            this.errorMessage = 'You are not part of any activated project.';
```

Comment:
'You are not a part of any active projects.'

Reply:
fixed in 3717fc5